glue/core/data_factories/dendro_loader.py

```diff
@@ -34,3 +36,69 @@ def load_dendro(file):
     im = Data(intensity=dg.data, structure=dg.index_map)
     im.join_on_key(dendro, 'structure', dendro.pixel_component_ids[0])
     return [dendro, im]
+
+
+def is_dendro(file, **kwargs):
+
+    if is_hdf5(file):
+
+        import h5py
+
+        f = h5py.File(file, 'r')
+
+        return 'data' in f and 'index_map' in f and 'newick' in f
+
+    elif is_fits(file):
+
+        from ...external.astro import fits
+
+        hdulist = fits.open(file)
+
+        # In recent versions of Astropy, we could do 'DATA' in hdulist etc. but
+        # this doesn't work with Astropy 0.3, so we use the following method
+        # instead:
+        try:
+            hdulist['DATA']
```

Comment:
No, this does not read any data from the disk, since ``fits.open`` defaults to using memory mapping, so it only reads data at the last possible minute.